src/uct/sm/mm/base/mm_iface.h

```diff
@@ -108,6 +108,10 @@ typedef struct uct_mm_fifo_ctl {
 
     /* 2nd cacheline */
     volatile uint64_t         tail;           /* How much was consumed */
+    struct {
+        pid_t                 pid;            /* Process owner pid */
+        ucs_time_t            starttime;      /* Process starttime */
```

Comment:
why need starttime both on ep and iface?

Reply:
for sysv/posix/xpmem owner of segment stores own starttime in fifo control segment, later EP uses this info to check process alive. starttime is copied into EP because in case of xpmem when iface process is terminated it makes fifo memory unavailable and access to this memory produces seg.fault.